packages/flutter_driver/lib/src/common/find.dart

```diff
@@ -119,17 +119,17 @@ class WaitUntilNoTransientCallbacks extends Command {
   String get kind => 'waitUntilNoTransientCallbacks';
 }
 
-/// A Flutter Driver command that waits until the frame is synced.
-class WaitUntilNoPendingFrame extends Command {
-  /// Creates a command that waits until there's no pending frame scheduled.
-  const WaitUntilNoPendingFrame({ Duration timeout }) : super(timeout: timeout);
+/// A Flutter Driver command that waits until the Flutter app is idle.
+class WaitUntilIdle extends Command {
```

Comment:
Any reason we do not expose this command (like the other commands) in the FlutterDriver API? e.g. https://github.com/flutter/flutter/blob/53186706de58544353a7bcbd16f631d2018bbba2/packages/flutter_driver/lib/src/driver/driver.dart#L475-L477

Reply:
Ya... Espresso can just call this extension API but if you think it makes sense to expose to the Driver API, happy to make that change in a follow-up PR. But we might want to sort out the relationship between this one and the waitUntilNoTransientCallbacks: shall we simply deprecate the waitUntilNoTransientCallbacks method? After we have this waitUntilIdle API, I don't see there are cases that users would want to use the waitUntilNoTransientCallbacks API.

Reply:
We should probably sort out that relationship before we add more API.